### DISCUSSION TITLE
Confused/newbie - can I use this to explore my local project?

### DISCUSSION THREAD

**Discussion by yatesco** [evcxr/evcxr#204] [Q&A] [answered]
I've been using it to try out various libraries via the CLI `evcxr`, and it works well. Can I also use this to explore a locally checked out crate/workspace? Like Scala's `sbt` or Clojure's `lein`?

(starting `evcxr` in my local workspace didn't seem to do anything different, hence my question)

Thanks!

**Comment by davidlattimore** [comment, marked as answer]
Sorry about the slow reply. You can call code from an external project, but you first need to import it with the `:dep` command.

For example:

```
:dep mylib = { path = "." }
mylib::foo();
```

Hope that helps.

**Reply by yatesco** [reply]
no problem - thanks for the answer. I should have thought of that as it is the most obvious thing! :-)
